src/common.ts

```diff
@@ -61,52 +50,26 @@ export const createAtoms = <
       const resultAtom = get(baseStatusAtom)
       return get(resultAtom)
     },
-    (get, set, action: Action) => {
-      const client = getClient(get)
-      const refresh = () => {
-        set(refreshAtom, (c) => c + 1)
-      }
-      return handleAction(action, client, refresh)
+    (_, __, action: Action | Partial<OperationContext>) => {
+      return handleAction(action)
     }
   )
 
-  const baseDataAtom = atom((get) => {
```

Comment:
baseDataAtom and dataAtom is the way to support suspense.
Can we keep the original structure?

Reply:
Unsure if it changes anything really as both status and data atoms are suspended currently.
Which is clearly seen in all tests even without this. Plus, unsure if old logic with filtering of all the signals depending on if there is error or not, is healthy. As I said in PR description, IMO I would get rid of data atom altogether as rather useless API.

Reply:
I'll test it out without data atom, so it will be only atomWithQuery/Subscription/Mutation. Will see if it changes anything. But now it gets suspended on the first load.
Weirdly enough it works even without a Suspense wrapper, just shows blank space before rendering entire thing, and I don't have any protection like save data access via optional chaining or guarding of any sort in the component, so from my understanding it should fail with "something is undefined" TypeError or similar, but it just suspends, but without any loading fallback (as if loading fallback is `() => null`, component)

Reply:
> Weirdly enough it works even without a Suspense wrapper

Maybe you use `atomWithObservable` without `initialData` for `statusAtom`.
It will trigger suspense for the initial load.